Update Dockerfile

AntoniaChroni opened the pull request:
Add R packages for fishplot analysis. Please test if it works.

Commit:
Update Dockerfile

Add R packages for fishplot analysis

Commit:
Update Dockerfile

Added packages for bulk-RNA-Seq deconvolution

AntoniaChroni commented:
Add R packages for immune-deconv module.

rjcorb reviewed (changes requested):
@AntoniaChroni I know we talked 1:1 but just documenting here that some of the new packages were not installed.

Commit:
Update Dockerfile

This PR adds GSVA and colorblindr packages to project docker image, and fixes install issues with ComplexHeatmap by installing via install_bioc.r.

This also includes additional R packages (purrr and plyr) and python (python3, pip3, and dependencies: SciPy, NumPy, Biopython) necessary for the clone phylogenies and fishplot module.

Commit:
Update Dockerfile

added version of python dependencies

AntoniaChroni commented:
This PR adds GSVA and colorblindr packages to project docker image, and fixes install issues with ComplexHeatmap by installing via install_bioc.r.

This also includes additional R packages (purrr and plyr) and python (python3, pip3, and dependencies: SciPy, NumPy, Biopython) necessary for the clone phylogenies and fishplot module.

Please pull latest docker image using the following command, and ensure that packages have been installed successfully:

docker pull pgc-images.sbgenomics.com/antoniachroni/pbta-tumor-evolution:latest

@rjcorb @jharenza

rjcorb reviewed (commented):
Can confirm that all packages load as expected. It looks like there are some main branch files and directories that are missing in this branch (`scripts/`, `download_data.sh`, etc.). I would run `git merge main` in the `dockerfile-v2` branch and push that commit, to ensure that the only difference between branches is the Dockerfile.

Commit:
Merge branch 'main' into dockerfile-v2

rjcorb reviewed (approved):
Looks good!
